packages/base/number.gts

```diff
@@ -33,6 +48,12 @@ function _serialize(val: number): string {
 export default class NumberCard extends CardBase {
   static [primitive]: number;
   static [useIndexBasedKey]: never;
+  static async [deserialize]<T extends CardBaseConstructor>(
+    this: T,
+    address: any
+  ): Promise<CardInstanceType<T>> {
+    return _deserialize(address).value as CardInstanceType<T>;
+  }
```

Comment:
I had to override this so that `createFromSerialized` would handle a bad input value like "some string" into a null. 

I notice this is up in hands as to how the developer will implement it.
_deserialize is supposed to be a UI thing for the input box. Whereas deserialize symbol on the card really handles the real model updating. Ideally they would be the same, but I think its wise to keep the option of not coupling them unless done manually like here